crates/algokit_transact_ffi/src/lib.rs

```diff
@@ -454,16 +456,16 @@ pub fn get_encoded_transaction_type(bytes: &[u8]) -> Result<TransactionType, Alg
     }
 }
 
-#[ffi_func] 
```

Comment:
Similar comment as I made on YAML RE formatting. We should define what tools we are using